hybris.c

```diff
@@ -30,26 +30,48 @@
 #define LIB_DROID_MEDIA_PATH "libdroidmedia.so"
 #endif
 
-void *android_dlopen(const char *name, int flags);
-void *android_dlsym(void *handle, const char *name);
+#ifndef LIBHYBRIS_PATH
+#define LIBHYBRIS_PATH "libhybris-common.so"
```

Comment:
`"libhybris-common.so.1"`

The unversioned .so isn't typically installed on devices and the lookup was failing for me.